fabtests/multinode/src/core_coll.c

```diff
@@ -202,7 +202,7 @@ static int barrier_test_run()
 	if (err) {
 		FT_DEBUG("barrier collective not supported: %d (%s)\n", err,
 			 fi_strerror(err));
-		return err;
+		return 0;
```

Comment:
Because I can't comment on the commit message...  Please update the message to have the commit title be self-contained and about 72 characters in length.  This helps with git log --oneline being used to identify commits that may need to be ported to stable branches.

This comment applies to several patches in this series.  The format for a commit message should be:
```
module/item: Start with upper case and state change

After a blank line enter a lot more details. Blah blah blah and
enter hard-coded line breaks around 80 characters in.  Follow with
another blank line before the signed-off-by

Signed-off-by: Me <me@me.org>
```

Reply:
Done